app/src/main/res/values/strings.xml

```diff
@@ -787,7 +787,7 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_leaf_type_needles">Needles</string>
     <string name="quest_leaf_type_broadleaved">Broadleaved</string>
     <string name="quest_leaf_type_mixed">Both are present</string>
-    <string name="create_new_note_hint">Best write the note in the locally spoken language or otherwise in English.</string>
+    <string name="create_new_note_hint">Ideally, write the note in the local language, or otherwise in English.</string>
```

Comment:
I think you could drop the first comma, although the Internet seems conflicted:
```suggestion
    <string name="create_new_note_hint">Ideally write the note in the local language, or otherwise in English.</string>
```

Reply:
I think that one should rather argue about the second comma. I would delete it because "otherwise in English" provides necessary information for the sentence and cannot stand on its own.

Reply:
Without looking at any prescriptive sources, I agree that the second comma is more debatable than the first, but I don't think either is a big deal either way.

Reply:
Using Grammarly for this sentence, my gut feeling was approved: The first comma should be there. For the second comma, I'd also go with @tapetis and remove it.

Reply:
I was debating was whether "otherwise" should be flanked by (appositive) commas. But it doesn't really matter, they're all fine enough.

```suggestion
    <string name="create_new_note_hint">Ideally, write the note in the local language or, otherwise, in English.</string>
```